packages/at_client/pubspec.yaml

```diff
@@ -27,7 +27,7 @@ dependencies:
   archive: ^3.3.5
   http: ^0.13.5
   internet_connection_checker: ^1.0.0+1	
-  async: ^2.10.0
+  async: ^2.9.0
```

Comment:
should be 2.9.0

Reply:
I disagree

Reply:
Updated

Reply:
I think I misunderstood the context and mistakenly thought Murali's suggestion was to remove the ^